src/core/cache-store.ts

```
@@ -387,45 +402,44 @@ export class CacheStore {
-          entriesToEvict
+          entriesToEvict,
         );
         break;
       case EvictionPolicy.LFU:
         keysToEvict = this.lfuPolicy.getEntriesToEvict(
           Array.from(this.cache.entries()),
-          entriesToEvict
+          entriesToEvict,
         );
         break;
       case EvictionPolicy.NONE:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)